vendor/modules.txt

```diff
@@ -1408,4 +1408,4 @@ sigs.k8s.io/yaml
 # k8s.io/cli-runtime => k8s.io/cli-runtime v0.20.0-beta.2
 # k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.20.0-beta.2
 # k8s.io/code-generator => k8s.io/code-generator v0.20.0-beta.2
-# github.com/gophercloud/gophercloud => github.com/gophercloud/gophercloud v0.11.0
```

Comment:
i think this change should be reverted in order to make the verify-gomod job pass